### PR TITLE
(node/lsstcam-vac.cp.lsst.org) deploy host as el9

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

**Review by glennmorris — APPROVED** [review]

File: hieradata/node/lsstcam-vac.cp.lsst.org.yaml
Comment (jhoblitt) [inline hieradata/node/lsstcam-vac.cp.lsst.org.yaml:1]: Please add this device to ipam.  A new location will likely need to be created.

File: hieradata/node/lsstcam-vac.cp.lsst.org.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    content:
+      connection:
+        id: "enp3s0"
+        uuid: "edb4ebb1-6d05-427b-b6ea-d483bdf32ec7"
+        type: "ethernet"
+        interface-name: "enp3s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "192.168.1.125/24"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-vac.cp.lsst.org.yaml]: If this is more than a point-to-point network, please add it to ipam.

File: spec/hosts/nodes/lsstcam-vac.cp.lsst.org_spec.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        it { expect(nm_keyfile['ipv4']['dns-search']).to eq('cp.lsst.org;') }
+      end
+
+      context 'with enp3s0' do
+        let(:interface) { 'enp3s0' }
+
+        it_behaves_like 'nm enabled interface'
+        it_behaves_like 'nm ethernet interface'
+        it { expect(nm_keyfile['ipv4']['address1']).to eq('192.168.1.125/24') }
+        it { expect(nm_keyfile['ipv4']['method']).to eq('manual') }
```
Comment (jhoblitt) [inline spec/hosts/nodes/lsstcam-vac.cp.lsst.org_spec.rb]:
```suggestion
        it_behaves_like 'nm manual interface'\
```

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: hieradata/node/lsstcam-vac.cp.lsst.org.yaml
Comment (dtapiacl) [reply hieradata/node/lsstcam-vac.cp.lsst.org.yaml:1]: New location was created, named: **CP-Camera**

File: hieradata/node/lsstcam-vac.cp.lsst.org.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/node/lsstcam-vac.cp.lsst.org.yaml]: Done.

File: hieradata/node/lsstcam-vac.cp.lsst.org.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    content:
+      connection:
+        id: "enp3s0"
+        uuid: "edb4ebb1-6d05-427b-b6ea-d483bdf32ec7"
+        type: "ethernet"
+        interface-name: "enp3s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "192.168.1.122/24"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-vac.cp.lsst.org.yaml:29]: This address is in ipam but it isn't associated with the device.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: hieradata/node/lsstcam-vac.cp.lsst.org.yaml (hunk 3 above)
Comment (dtapiacl) [reply hieradata/node/lsstcam-vac.cp.lsst.org.yaml:29]: Done

**Review by jhoblitt — APPROVED** [review]
LGTM. Thank you for updating ipam.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Please squash 70221be54e5f89c2744a943c244ef028959c1409

**Comment by dtapiacl** [comment]
> Please squash [70221be](https://github.com/lsst-it/lsst-control/commit/70221be54e5f89c2744a943c244ef028959c1409)

Commit was squashed on to: [0d46834](https://github.com/lsst-it/lsst-control/pull/1150/commits/0d46834937e2d92802802f0d80d5e7e2f3789ae6)

**Review by jhoblitt — APPROVED** [review]

**Pull request merged**